src/raftstore/store/peer.rs

```diff
@@ -957,36 +957,27 @@ impl Peer {
 
     pub fn post_apply(
         &mut self,
-        res: &ApplyRes,
         groups: &mut HashSet<u64>,
-        store_stat: &mut StoreStat,
+        apply_state: RaftApplyState,
```

Comment:
Why change the signature?

Reply:
Because `on_ready_results` needs some fileds of the `ApplyRes`, so `res` is partially moved before `post_apply`.